libs/SmartSync/src/com/salesforce/androidsdk/smartsync/target/SyncUpTarget.java

```diff
@@ -157,7 +174,7 @@ public String createOnServer(SyncManager syncManager, JSONObject record, List<St
      * @throws IOException
      * @throws JSONException
      */
-    public String createOnServer(SyncManager syncManager, String objectType, Map<String, Object> fields) throws IOException, JSONException {
+    protected String createOnServer(SyncManager syncManager, String objectType, Map<String, Object> fields) throws IOException, JSONException {
```

Comment:
Same story as deleteOnServer (used to be called from manager but now is called from createOnServer(syncManager, record, fieldlist).